packages/devextreme/js/__internal/ui/chat/chat_message_box.ts

```diff
@@ -37,10 +58,44 @@ class MessageBox extends Widget<any> {
       .addClass(CHAT_MESSAGE_BOX_BUTTON_CLASS)
       .appendTo(this.element());
 
-    this._button = this._createComponent($button, Button, {
+    const configuration: ButtonProperties = {
       icon: 'send',
       stylingMode: 'text',
-    });
+      onClick: (e): void => {
+        this._buttonClickHandler(e);
+      },
+    };
+
+    this._button = this._createComponent($button, Button, configuration);
+  }
+
+  _initSendButtonClickAction(): void {
+    this._sendButtonClickAction = this._createActionByOption('onSendButtonClick');
+  }
+
+  _buttonClickHandler(e: ClickEvent): void {
+    // @ts-expect-error
+    // eslint-disable-next-line no-unsafe-optional-chaining
+    const { text } = this._textArea?.option();
+
+    if (!text) {
+      return;
+    }
+
+    this._sendButtonClickAction({ text, event: e.event });
+    this._textArea?.option({ value: '' });
```

Comment:
Can we use reset() method? https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxTextArea/Methods/#reset

Reply:
[151eeb6b38743d0dd850079836ba81a6e99396d3]